base/sort.jl

```diff
@@ -1569,6 +1569,12 @@ The algorithms used by `DEFAULT_UNSTABLE` are an implementation detail. They are
 the same as those used by [`DEFAULT_STABLE`](@ref), but this is subject to change in future.
 """
 const DEFAULT_UNSTABLE = DEFAULT_STABLE
+"""
+    SMALL_THRESHOLD
+    
+    SMALL_THRESHOLD is constant for internally decide, whatever is arrey considered small or big
```

Comment:
A few pointers:

This has spelling errors and is not grammatically correct (in fact I have trouble figuring out what it tries to say).

In addition: this line shouldn't be indented, there should be an empty line before this docstring, and it shouldn't end in an empty line.

Reply:
Ok I will fix it when I get to the computer

Reply:
Its better now?